qiita_db/processing_job.py

```diff
@@ -281,20 +336,28 @@ def heartbeat(self):
             qdb.sql_connection.TRN.add(sql, [self.id])
             return qdb.sql_connection.TRN.execute_fetchlast()
 
-    @heartbeat.setter
-    def heartbeat(self, value):
-        """Sets the timestamp for the last received heartbeat
+    def execute_heartbeat(self):
```

Comment:
Can you provide a better name? I didn't like it either, but I couldn't come up with a good name.